utils/python/CIME/hist_utils.py

```diff
@@ -282,6 +302,16 @@ def generate_baseline(case, baseline_dir=None):
             if os.path.exists(baseline):
                 os.remove(baseline)
 
+            #special care for multi-instance cases,
```

Comment:
For multiinstance only keep the first instance in the baseline and rename it to remove the _0001